actors/evm/src/interpreter/stack.rs

```diff
@@ -78,7 +78,9 @@ impl Stack {
     }
 
     #[inline]
+    // Panics if `i == 0`.
```

Comment:
I'd expand on this and write some proper documentation. I.e.:

- Tell the user what this does.
- Explain _why_ i == 0 panics.